src/ai/backend/manager/api/gql/scaling_group/resolver.py

```diff
@@ -0,0 +1,82 @@
+"""GraphQL query resolvers for scaling group system."""
+
+from __future__ import annotations
+
+from typing import Optional
+
+import strawberry
+from strawberry import Info
+from strawberry.relay import Connection, Edge
+
+from ai.backend.manager.api.gql.base import to_global_id
+from ai.backend.manager.services.scaling_group.actions.list_scaling_groups import (
+    SearchScalingGroupsAction,
+)
+
+from ..types import StrawberryGQLContext
+from .types import ScalingGroupFilter, ScalingGroupOrderBy, ScalingGroupV2
+
+# Connection types
+
+ScalingGroupEdge = Edge[ScalingGroupV2]
+
+
+@strawberry.type(description="Added in 25.18.0. Scaling group connection")
+class ScalingGroupV2Connection(Connection[ScalingGroupV2]):
+    count: int
+
+    def __init__(self, *args, count: int, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.count = count
+
+
+# Query fields
+
+
+@strawberry.field(description="Added in 25.18.0. List scaling groups")
+async def scaling_groups(
```

Comment:
Is this scaling_groups to be used as a whole? Does it overlap with the existing query?